appengine-java8/mail/pom.xml

```diff
@@ -46,6 +46,12 @@ Copyright 2016 Google Inc.
       <artifactId>mail</artifactId>
       <version>1.4.7</version>
     </dependency>
+      <dependency>
+          <groupId>com.google.appengine</groupId>
+          <artifactId>appengine-api-1.0-sdk</artifactId>
+          <version>RELEASE</version>
```

Comment:
Please change from `RELEASE` to the latest version number instead (for all dependencies). 

We have a tool that incrementally updates to ensure the samples are working correctly (it's not running for this project because it isn't linked in the main pom.)